data/json/recipes/food/bread.json

```diff
@@ -168,7 +168,7 @@
     "activity_level": "LIGHT_EXERCISE",
     "result": "buttered_toast_wheat_free",
     "copy-from": "toast_wheat_free",
```

Comment:
because this recipe copies from toast_wheet_free, it currently requires heat and a toasting surface. It's correct for it to call for bread, not toast: you're starting with bread, toasting it, and buttering it. Makign it require toast means you're toasting the bread twice.

Reply:
Finally understood what you were getting at. The original toast w/ spread recipes were inconsistent with whether they wanted bread or toast, so I decided to stick with using toast for them, especially since the cooking requirements weren't showing up.

Now that I've got that sorted, it seems the original intention was for these toast w/ spread recipes to start with plain bread and toasting it in the process.